### PR TITLE
[docs] Add link to the flow.deckhouse.io on the releases page

### PR DESCRIPTION
**Pull request by z9r5**
## Description
Add link to the flow.deckhouse.io on the releases page.

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries
```changes
section: docs
type: chore
summary: Add link to the flow.deckhouse.io on the releases page.
impact_level: low
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

File: docs/documentation/pages/RELEASES.md [hunk 1]
```diff
@@ -4,18 +4,18 @@ permalink: en/deckhouse-release-channels.html
...
-<p>Clusters, as elements of the infrastructure, usually have different requirements.</p>
-<p>A production cluster, unlike a development cluster, has higher reliability requirements. Frequent updates and changes to components are undesirable on a productive cluster. Components should be tested as much as possible.
-</p>
-We use <b>five release channels</b>.
-</div>
+Clusters, as elements of the infrastructure, usually have different requirements.
+
+A production cluster, unlike a development cluster, has higher reliability requirements. Frequent updates and changes to components are undesirable on a productive cluster. Components should be tested as much as possible.
+
+Deckhouse uses **five release channels**, between which it can "soft" switch using the [deckhouse](modules/002-deckhouse/) module: it is enough to specify the desired release channel in the module [configuration](modules/002-deckhouse/configuration.html#parameters-releasechannel).
```
Comment (kirkonru) [inline docs/documentation/pages/RELEASES.md:18]:
```suggestion
Clusters, as infrastructure elements, usually have to meet various requirements.

A production cluster, unlike a development one, has higher requirements for reliability. In a production cluster, frequent component updates and changes are undesirable. All the cluster components must be thoroughly tested for stable and reliable operation.

Deckhouse uses **five release channels** which you can *soft-switch* between using the [deckhouse](modules/002-deckhouse/) module: just specify the desired release channel in the module [configuration](modules/002-deckhouse/configuration.html#parameters-releasechannel).
```

File: docs/documentation/pages/RELEASES_RU.md [hunk 2]
```diff
@@ -5,18 +5,19 @@ layout: page
...
 ---
-{::options parse_block_html="false" /}
-{% asset releases.css %}
+
+<link rel="stylesheet" type="text/css" href='{{ assets["releases.css"].digest_path | true_relative_url }}' />
 {%- assign releases = site.data.releases.channels | sort: "stability" -%}
 
-<div class="page__container page_releases">
+<div class="docs__information warning active">
+Информацию о том, какие версии Deckhouse находятся в настоящий момент на каких каналах обновлений, а также информацию о планируемой дате смены версии на канале обновлений, смотрите на сайте <a href="https://flow.deckhouse.io" target="_blank">flow.deckhouse.io</a>.
```
Comment (kirkonru) [inline docs/documentation/pages/RELEASES_RU.md:13]:
```suggestion
Информацию о том, какие версии Deckhouse находятся в настоящий момент на каких каналах обновлений, а также о планируемой дате смены версии на канале обновлений смотрите на сайте <a href="https://flow.deckhouse.io" target="_blank">flow.deckhouse.io</a>.
```

File: docs/documentation/pages/RELEASES_RU.md [hunk 3]
```diff
@@ -5,18 +5,19 @@ layout: page
...
-<p>К кластерам как элементам инфраструктуры обычно предъявляются различные требования.</p>
-<p>Например, production-кластер, в отличие от кластера разработки, имеет более высокие требования к надежности, на нем нежелательно часто обновлять или изменять какие-либо компоненты без особой необходимости, а сами компоненты должны быть максимально протестированы.
-</p>
-Чтобы покрыть самые частые случаи организации окружений, а также с целью повысить качество самого Deckhouse, мы используем <b>пять каналов обновлений</b>.
-</div>
+К кластерам как элементам инфраструктуры обычно предъявляются различные требования.
+
+Например, production-кластер, в отличие от кластера разработки, имеет более высокие требования к надежности: на нем нежелательно часто обновлять или изменять какие-либо компоненты без особой необходимости, а сами компоненты должны быть максимально протестированы.
+
+Deckhouse использует **пять каналов обновлений**, между которыми может *мягко* переключаться с помощью модуля [deckhouse](modules/002-deckhouse/): достаточно указать желаемый канал обновлений в [конфигурации](modules/002-deckhouse/configuration.html#parameters-releasechannel) модуля.
```
Comment (kirkonru) [inline docs/documentation/pages/RELEASES_RU.md:20]:
```suggestion
К кластерам как элементам инфраструктуры обычно предъявляются различные требования.

Например, production-кластер, в отличие от кластера разработки, более требователен к надежности: в нем нежелательно часто обновлять или изменять какие-либо компоненты без особой необходимости, при этом сами компоненты должны быть тщательно протестированы.

Deckhouse использует **пять каналов обновлений**. *Мягко* переключаться между ними можно с помощью модуля [deckhouse](modules/002-deckhouse/): достаточно указать желаемый канал обновлений в [конфигурации](modules/002-deckhouse/configuration.html#parameters-releasechannel) модуля.
```

**Review by kirkonru — DISMISSED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [4083](https://github.com/deckhouse/deckhouse/pull/4083) to the branch [release-1.44](https://github.com/deckhouse/deckhouse/tree/release-1.44) successful!

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [4086](https://github.com/deckhouse/deckhouse/pull/4086) to the branch [release-1.43](https://github.com/deckhouse/deckhouse/tree/release-1.43) successful!

**Pull request merged**